streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyKeyValueStore.java

```diff
@@ -65,9 +66,10 @@
      * Order is not guaranteed as bytes lexicographical ordering might not represent key order.
      *
      * @param from The first key that could be in the range, where iteration ends.
+     *             A null value indicates that the range ends with the first element in the store.
      * @param to   The last key that could be in the range, where iteration starts from.
+     *             A null value indicates a starting position from the last element in the store.
```

Comment:
It looks like these are backwards. If I understand this method signature, even though iteration is reversed, the "from" key is still the lower bound, and the "to" key is the upper.